src/Equation.jl

```diff
@@ -250,7 +250,12 @@ function string_tree(
             end
         end
     elseif tree.degree == 1
-        return "$(options.unaops[tree.op])($(string_tree(tree.l, options, bracketed=true, varMap=varMap)))"
+        if occursin("_nan", string(options.unaops[tree.op]))
```

Comment:
It might be good to add a test for this function, so that `log_nan` is converted to, e.g., `"log"`.

Also I'm not sure if it should be this general, as a user could define their own function using `"_nan"` in the title - we wouldn't want to convert that. Maybe this could be a dictionary mapping for the internal operators?

Reply:
Sure, I can use a dictionary here instead. I sort of just assumed that `_nan` would be a rare enough name, but I agree.